README.md

```diff
@@ -53,3 +56,16 @@ This will allow you to view the site on a web browser at `http://localhost:8000/
 Make your changes and test in your local environment. Then submit a PR to this repo.
 
 ### ❓ Got a question, something to discuss or just curious to learn more about this project? Check out the [DISCUSSION](../../discussions) section.
+
+## Contributors ✨
```

Comment:
@Mogakamo I'm not able to see the list of contributors to the repo following the change you made. Can you please check what is required to correctly use the allcontributors feature? You can read the docs at:  https://allcontributors.org/docs/en/overview
Thanks for your efforts!

Reply:
I am not able to get all the contributors on the. README.md file since I do not have write access on the repo but I have set a script that I had set to help you add it since you are the one who has write access

Reply:
@funbeedev I will be sharing the command to be able to do so.
That section on the readme for contributors is updated automatically so it should not be modified manually

Reply:
@Mogakamo Please share the command or make the updates so the README will show the contributors. Thanks!

Reply:
@funbeedev you can use the command above to add a contributor. For now, you can do it from this pr but as soon as you merge it you can do it from the main branch.
Incase of anthing just tag me